### PR TITLE
add 'Cruz Roja' wikidata & wikipedia

### PR DESCRIPTION
**Pull request by adrcav**
I added wikidata and wikipedia data for Cruz Roja in brands/amenity/hospital.

### CODE REVIEW

File: data/brands/amenity/hospital.json [hunk 1]
```diff
@@ -36,10 +36,12 @@
     {
       "displayName": "Cruz Roja",
       "id": "cruzroja-abc639",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["ch"]},
```
Comment (Binnette) [inline data/brands/amenity/hospital.json:39]: Hi. "ch" stands for Switzerland. You should replace "ch" by "es". "es" stands for Spain. 

Details : https://en.wikipedia.org/wiki/List_of_countries_by_United_Nations_geoscheme

[the remark this answers is not in the document]
Comment (Binnette) [reply data/brands/amenity/hospital.json:39]: And BTW "ch" is Switzerland ;)

**Review by Binnette — CHANGES_REQUESTED** [review]
Hello

File: data/brands/amenity/hospital.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (adrcav) [reply data/brands/amenity/hospital.json]: Yes, you're right. I've updated it.

**Comment by UKChris-osm** [comment]
If "_Cruz Roja_" is Spanish for "_Red Cross_" would it be better to target the locationSet to Spanish speaking countires?

**Review by bhousel — CHANGES_REQUESTED** [review]
> If "_Cruz Roja_" is Spanish for "_Red Cross_" would it be better to target the locationSet to Spanish speaking countires?

I agree we don't want _Cruz Roja_ worldwide.  Just include the parts of the world where it is called that.

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks @adrcav , looks good 👍

**Pull request merged**